starknet_py/net/client_models.py

```diff
@@ -71,8 +71,28 @@ class L2toL1Message:
     """
 
     payload: List[int]
-    l1_address: int
-    l2_address: int
+    l1_address: int  # from_address in spec
+    l2_address: int  # to_address in spec
```

Comment:
If it's L2 -> L1 message, shouldn't `from_address` be `l2_address`?

Reply:
Yup, thanks, good catch!